bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBaseThingHandler.java

```diff
@@ -46,23 +65,59 @@
     private HashMap<String, Integer> typeTable = new HashMap<>();
     protected HashMap<String, String> stateNames = new HashMap<>();
 
+    protected String url = "";
+
+    private Map<String, Unit<?>> units = new HashMap<>();
+
     public SomfyTahomaBaseThingHandler(Thing thing) {
         super(thing);
+        // Define default units
+        units.put("Number:Temperature", SIUnits.CELSIUS);
+        units.put("Number:Energy", Units.WATT_HOUR);
+        units.put("Number:Illuminance", Units.LUX);
+        units.put("Number:Dimensionless", Units.PERCENT);
     }
 
     public HashMap<String, String> getStateNames() {
         return stateNames;
     }
 
-    protected String url = "";
-
     @Override
     public void initialize() {
-        url = getURL();
-        if (getThing().getProperties().containsKey(RSSI_LEVEL_STATE)) {
-            createRSSIChannel();
+        logger.debug("initializing thing handler for thing {}", getThing().getUID());
```

Comment:
Can this message be replaced by using the debugger or by increasing the framework's log level? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging

This is already logged [here](https://github.com/openhab/openhab-core/blob/e20348e8d66845db841f17e6e048d789d004fdf2/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ThingManagerImpl.java#L815) and can therefore be removed.

Reply:
I changed into `logger.trace`.

Reply:
The log message is redundant and should therefore be removed. You could set the log level of `org.openhab.core.thing` to `DEBUG` to see the log message from the framework: "Calling initialize handler for thing '{}' at '{}'."

Reply:
Log removed